Scripts/Action/BooleanAction.cs

```diff
@@ -30,52 +30,52 @@ public class BooleanActionUnityEvent : UnityEvent<bool, object>
         public BooleanActionUnityEvent Deactivated = new BooleanActionUnityEvent();
 
         /// <inheritdoc />
-        public override void Receive(bool value, object sender = null)
+        public override void Receive(bool value, object initiator = null)
         {
             previousValue = Value;
             Value = value;
             State = Value;
             if (State)
             {
-                OnActivated(true);
+                OnActivated(true, initiator);
             }
             else
             {
-                OnDeactivated(false);
+                OnDeactivated(false, initiator);
             }
 
             if (HasChanged())
             {
-                OnChanged(value);
+                OnChanged(value, initiator);
             }
         }
 
         /// <inheritdoc />
-        protected override void OnActivated(bool value)
+        protected override void OnActivated(bool value, object sender)

```

Comment:
change to `initiator` like done in other places

Reply:
or are you having the `On` methods use `sender` but everything else use `initiator`?